src/main/resources/db/migration/V2017_09_12_1315__add_colums_for_ccj.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE claim
+  ADD COLUMN county_court_judgment JSONB,
```

Comment:
we sure we don't want another table, even if it is a 1:1 mapping?

Reply:
I have talked to @geek-soft  about it. Yes, we are sure we want to have it in claim table that is going to be eventually rename to case with structure (not final design - just overview):

```
case:
   id
   externalId
   claimantId
   defendantId
   claim
   response
   ccj
   createdAt
   responsedAt
   ccj'dAt
```

Reply:
it can still be normalised so that not everything is in one table.
but sure.

Reply:
In my opinion if we are going to migrate to CCD it's good to keep it as simple as possible. IMO LEFT JOIN is not "as simple as possible" ;) What's more in ccj table we would duplicate 3/5 columns from claim.

But to make it better we should redefine our dictionary:
claim -> case
claimData -> claim